.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: Go
+
+on:
+  push:
```

Comment:
Please change the target on push event.
This implementation only supports this branch, but ci should support all pull request.